js/map.js

```diff
@@ -89,12 +89,49 @@ const renderAdsOnMap = (ads) => {
       );
 
       secondaryMarker
-        .addTo(map)
+        .addTo(markers)
         .bindPopup(
           createCustomPopup(author, offer),
         )
     })
 };
 
+const getFilteredAds = (ads, typeHouses) => {
+  let filteredAds = ads;
+  if (typeHouses) {
+    filteredAds = ads.filter(ad => ad.offer.type.includes(typeHouses));
+  }
+  return filteredAds;
+}
+
+let localAds = []
```

Comment:
```suggestion
const localAds = []
```
Перенести это на верх модуля